ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -38,15 +38,15 @@ export BITSTREAM="--offline --list ${SHA}"
 
 # We will lose serial access when we reboot, but if tests fail we should reboot
 # in case we've crashed the UART handler on the CW310's SAM3U
-trap 'ci/bazelisk.sh run //sw/host/opentitantool -- --rcfile= --interface=cw310 fpga reset-sam3x' EXIT
+trap 'ci/bazelisk.sh run //sw/host/opentitantool -- --rcfile= --interface=cw310 fpga reset-sam3x --exec="fpga clear-bitstream"' EXIT
```

Comment:
Nit: I don't think this is necessary, better to keep this script simpler and cleanup at the start.

Reply:
Good point, I added this first, then the one at the start :)